pkgs/by-name/bi/bitrise/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "bitrise";
+  version = "2.19.0";
+
+  src = fetchFromGitHub {
+    owner = "bitrise-io";
+    repo = "bitrise";
+    rev = version;
+    hash = "sha256-VjuDeRl/rqA7bdhn9REdxdjRon5WxHkFIveOYNpQqa8=";
+  };
+
+  doCheck = false; # many tests rely on writing to $HOME/.bitrise
+
+  vendorHash = null;
+  ldflags = [
+    "-X github.com/bitrise-io/bitrise/version.Commit=${src.rev}"
+    "-X github.com/bitrise-io/bitrise/version.BuildNumber=0"
+  ];
+  CGO_ENABLED = 0;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    changelog = "https://github.com/bitrise-io/bitrise/releases/tag/${src.rev}";
```

@Pandapip1 is there anything solved about parametric changelogs?

---

I'd recommend doing

```suggestion
    changelog = "https://github.com/bitrise-io/bitrise/releases";
```

as this will display the full changelog instead of the abbreviated version that only shows the changes for a specific release.